requirements.txt

```
@@ -1,3 +1,3 @@
 ops==2.14.0
 distro==1.9.0
-pycryptodome==3.20.0
+pycryptodome==3.20.0
```

[nit]: Missing newline with the addition of `pycryptodome` as dependency. Weird that it's both adding and removing the line in diff?

[foresight]: we'll want to spend some time investigating next pulse using `poetry` for our charms after we reassemble the monorepo if we have the time.

```suggestion
pycryptodome==3.20.0

```

---

Added the newline